Content-type: text/xml

<?xml version="1.0" encoding="utf-8"?>
<testing-report run-id="12" judge-id="13" status="SE" scoring="ACM" run-tests="1" contest-id="3200" real-time-available="yes" max-memory-used-available="yes" correct-available="yes" failed-test="1" time-limit-ms="2000" real-time-limit-ms="5000" marked-flag="no" >
  <uuid>1abd31de-6908-4bdf-8576-0ea10a452bd1</uuid>
  <host>localhost.localdomain</host>
  <cpu-model>Intel(R) Pentium(R) CPU 3550M @ 2.30GHz</cpu-model>
  <cpu-mhz>2294.738</cpu-mhz>
  <compiler_output></compiler_output>
  <tests>
    <test num="1" status="SE" time="286" real-time="308" max-memory-used="18698240" >
  <program-stats-str>{ utime=259, stime=27, ptime=286, rtime=308, maxvsz=18698240, maxrss=19427328, nvcsw=1, nivcsw=38 }</program-stats-str>
      <input size="4">179
</input>
      <output size="0"></output>
      <stderr size="284">Can't open '/usr/share/abrt/conf.d/plugins/python3.conf': Operation not permitted
Can't open '/etc/abrt/plugins/python3.conf': Operation not permitted
Traceback (most recent call last):
  File &quot;./000012&quot;, line 4, in &lt;module&gt;
    a=int(myat(179))
NameError: name 'myat' is not defined
</stderr>
    </test>
  </tests>
</testing-report>
